### PR TITLE
fix: add info for problem 54 - project Euler

### PR DESCRIPTION
**Pull request by AdrianMuntean**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

[Problem 54](https://www.freecodecamp.org/learn/coding-interview-prep/project-euler/problem-54-poker-hands) has a vague input description. It doesn't mention the test case. It states the problem as a whole, but it's missing the info about the test case.

### CODE REVIEW

**Review by naomi-lgbt — APPROVED** [review]
LGTM~! 🥳 Congratulations on your first pull request!

As a note: our workflow requires at least one additional approval before this can be merged.

**Review by RandellDawson — APPROVED** [review]

**Comment by RandellDawson** [comment]
Congrats @AdrianMuntean on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 📝

**Pull request merged**
